tests/foreman/cli/test_rhcloud_iop.py

```diff
@@ -0,0 +1,262 @@
+"""CLI tests for IoP
+
+:Requirement: RHCloud
+
+:CaseAutomation: Automated
+
+:CaseComponent: RHCloud
```

Comment:
Change case component here to `Insights-Advisor`. We are doing this for Devendra to create a separate pipeline for IOP